mono/mini/interp/interp-internals.h

```diff
@@ -181,7 +181,11 @@ typedef struct _StackFragment StackFragment;
 struct _StackFragment {
 	guint8 *pos, *end;
 	struct _StackFragment *next;
-	double data [1];
+#if SIZEOF_VOID_P == 4
+	/* Align data field to MINT_VT_ALIGNMENT */
+	gint32 pad;
+#endif
```

Comment:
I assume this is because double is 4 byte aligned on x86 ?

Reply:
yes. And mono_64bitaligned_t doesn't work. We probably get alignment correctly with it by chance